pkg/utils/utils_test.go

```diff
@@ -0,0 +1,21 @@
+package utils
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFormatSize(t *testing.T) {
+	assert := assert.New(t)
+	kvs := map[int64]string{
+		1024:          "1.00 KB",
+		1024000:       "1000.00 KB",
+		1024000000000: "953.67 GB",
```

Comment:
I am wondering if we could add more test cases: negative ones, 0, and positive ones.

Reply:
yes, has completed cases